workers/docker-worker/src/lib/upload_to_s3.js

```diff
@@ -40,12 +42,21 @@ module.exports = async function uploadToS3 (
       await tmp.writeFile(source);
     } else {
       let stream = fs.createWriteStream(tmp.path);
-      await pipe(source, stream);
+      // TODO: Do we care if this is larger than the original?
```

Comment:
I suspect we don't care. I noticed this in tests where the original file is very small (5 bytes) and the resulting compressed file is 25 bytes.